app/views/complaints/_tta_report_list.html.erb

```diff
@@ -0,0 +1,14 @@
+ <div class="ct-section-divider">
+  <div class="display-flex flex-justify flex-align-center">
+    <span>
+      <p>
+        TTA Activity:
+        <% tta_reports.each_with_index do |report, i| %>
```

Comment:
[nit] Not sure you need `each_with_index` since you aren't using `i`, right?

Reply:
oh yeah i was gonna do a complicated thing initially with`,` but then laura just told me i could do one per line